airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -354,6 +357,16 @@ private static void assertColumnsWithSameNameAreSame(String schemaName, String t
                   return new ColumnInfo(f.get(INTERNAL_COLUMN_NAME).asText(), jdbcType);
                 })
                 .collect(Collectors.toList())))
+        .peek(t -> {
```

Comment:
I think this would be better done in a for loop after the sequence of stream operations for readability